doc/source/whatsnew/v0.19.0.txt

```diff
@@ -673,6 +673,7 @@ Performance Improvements
 Bug Fixes
 ~~~~~~~~~
 
+- Bug in ``pd.read_csv()`` causing a segfault when iterating over a large file in chunks (:issue:`13703`)
```

Comment:
can you elaborate a bit, explaining generally under what circumstances. A user reading this would be very alarmed, but this is a quite rare situation.